docs/guide/docs/main/overview.md

```diff
@@ -79,34 +114,44 @@ botpress-server
 ├── bp / bp.exe
 │
 └── data
-│   │
-│   ├── bots
-│   │   └── name-of-your-bot
-│   │       ├── bot.config.json # bot-specific configuration
-│   │       ├── actions <1>
-│   │       │   └── custom-code-1.js
-│   │       │   └── # etc...
-│   │       ├── flows
-│   │       └── # etc... mostly auto-generated by the Studio UI
-│   │
-│   └── global
-│       ├── actions <1> # shared actions across all bots
-│       ├── hooks <2> # lifecycle hooks (custom code)
-│       ├── botpress.config.json # configuration across all bots
-│       └── # etc... mostly auto-generated by installed modules
+│ │
+│ ├── bots
+│ │ └── name-of-your-bot
+│ │ ├── bot.config.json # bot-specific configuration
+│ │ ├── revisions.json # changelog of bot configurations
+│ │ ├── actions <1>
+│ │ │ └── custom-code-1.js
+│ │ │ └── # etc...
+│ │ ├── flows
+│ │ └── # etc...(content elements, config, hooks, intents, entities, media, modules, qna)
+│ │
+│ |── global
+│ | ├── actions <1> # shared actions across all bots
+│ | ├── hooks <2> # lifecycle hooks (custom code)
+│ | ├── botpress.config.json # configuration across all bots
+│ | └── # etc... mostly auto-generated by installed modules
+| |
+│ └── assets
+│ ├── modules <1> # shared actions across all bots
+│ ├── ui-admin <2> # lifecycle hooks (custom code)
+│ ├── ui-studio # configuration across all bots
+│ └── # etc... mostly auto-generated by installed modules
 │
 └── modules
-    ├── nlu.tgz
-    ├── webchat.tgz
-    └── # etc..
+├── nlu.tgz
+├── webchat.tgz
+└── # etc..
 ```
 
 ## Modules
 
-A module is an extra component outside of the Botpress Core itself that you can install in your bot to add new features to Botpress. Every bot uses modules in a way since almost everything in Botpress is a module. This architecture allows for maximum flexibility and freedom for developers.
+A module is an extra component outside of the Botpress Core itself that you can install in your bot to add new features to Botpress. Every bot uses modules in a way since almost everything in Botpress is a module. 
+
+This architecture allows for maximum flexibility and freedom for developers. It also ensures that any customizations are applied without affecting the smooth running of the Botpress Core.
```

Comment:
We should say that developers can build custom modules and link to the custom module section

Reply:
Done